### PR TITLE
PS-7115 Multistore - do not show products that are not assigned to the store

### PR DESCRIPTION
**Pull request by Compixx**
Epic ticket: https://spryker.atlassian.net/browse/PS-3812
Story ticket: https://spryker.atlassian.net/browse/PS-3814
Bug: https://spryker.atlassian.net/browse/PS-7115

### CODE REVIEW

File: src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php [hunk 1]
```diff
@@ -51,6 +51,43 @@ public function getEventTransferForeignKeys(array $eventTransfers, $foreignKeyCo
...
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\EventEntityTransfer[] $eventTransfers
+     * @param string $foreignKeyColumnName
+     * @param string $relatedForeignKeyColumnName
+     *
+     * @return array
+     */
+    public function getGroupedEventTransferRelatedForeignKeys(array $eventTransfers, $foreignKeyColumnName, string $relatedForeignKeyColumnName)
```
[the remark this answers is not in the document]
Comment (Compixx) [reply src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php]: I don't see acceptable way to do that. All variables are closely related if I move for example getting foreign key to a new function, I had to send there a lot of variables and also result array should be send there and received back in each call

[the remark this answers is not in the document]
Comment (vitaliykirichenkoit) [reply src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php]: Okay. Then please add missing type constraint to `$foreignKeyColumnName` parameter.

File: src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilterInterface.php [hunk 2]
```diff
@@ -31,4 +31,13 @@ public function getEventTransferForeignKeys(array $eventTransfers, $foreignKeyCo
...
     public function getEventTransfersByModifiedColumns(array $eventTransfers, array $columns);
+
+    /**
+     * @param \Generated\Shared\Transfer\EventEntityTransfer[] $eventTransfers
+     * @param string $foreignKeyColumnName
+     * @param string $relatedForeignKeyColumnName
+     *
+     * @return array
+     */
+    public function getGroupedEventTransferRelatedForeignKeys(array $eventTransfers, $foreignKeyColumnName, string $relatedForeignKeyColumnName);
```
Comment (vitaliykirichenkoit) [inline src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilterInterface.php]: Please add missing type constraint to $foreignKeyColumnName parameter.

**Comment by vitaliykirichenkoit** [comment]
Also, please merge latest master branch, since you have conflicting `EventBehaviorFacade` files.

**Pull request merged**